lib/dotcom/trip_plan/location_nudger.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Dotcom.TripPlan.LocationNudger do
```

Comment:
Open to suggestions for better names than this.

Reply:
`LocationOSMReconciler` ?

Reply:
How about `OpenStreetMapReconciler`?

Reply:
I like it!